docs/configs/worker_pinning.py

```diff
@@ -0,0 +1,28 @@
+# fmt: off
+
+from parsl.providers import LocalProvider
+
+from funcx_endpoint.endpoint.utils.config import Config
+from funcx_endpoint.executors import HighThroughputExecutor
+
+config = Config(
+    executors=[
+        HighThroughputExecutor(
+            max_workers_per_node=4,
+
+            # Each worker launched will have env vars set to one of (0..3)
+            available_accelerators=4,
+
+            # Alternatively a list of strings may be supplied
+            # available_accelerators=['opencl:gpu:1', 'opencl:gpu:2']
```

Comment:
Do I interpret that the second comment (lines 16-17) is related to the `available_accelerators` argument?  Perhaps move it so it's co-located then?
```diff
+ # `available_accelerators` may be a natural number or a list of strings.
+ # If an integer, then each worker launched will have an automatically
+ # generated environment variable.  In this case, one of 0, 1, 2, or 3.
+ # Alternatively, specific strings may be utilized.
 available_accelerators=4,
+ # available_accelerators=['opencl:gpu:1', 'opencl:gpu:2']  # alternative

-# Alternatively a list of strings may be supplied
-# available_accelerators=['opencl:gpu:1', 'opencl:gpu:2']
```
And leave a line between them and the following "unrelated" option.